Makefile

```diff
@@ -443,6 +445,7 @@ set-manifest-image:
 ## --------------------------------------
 
 RELEASE_TAG := $(shell git describe --abbrev=0 2>/dev/null)
+RELEASE_ALIAS_TAG ?= $(PULL_BASE_REF)
```

Comment:
nit: is this necessary given that we are always overriding the value in L527?

Reply:
Yes, this is used for regular release-staging. For the daily, I created a custom tag in L527.